lib/form/paragraph.flow

```diff
@@ -570,8 +570,8 @@ breakTextFragment(
 		}
 	};
 
-	makeTextFn = \indexing, t -> {
-		toInspectingElementIfNecessary(indexing, indexing, GeneralText(t))
+	makeTextFn = \indexing, t, s -> {
```

Comment:
this is bad. it will double every word, even if it doesn't have any hyphenation points